### ISSUE TITLE
Provide API to customize bucket boundaries for individual value recorders

### ISSUE THREAD

**Issue by fabian-braun** [open-telemetry/opentelemetry-rust#673]
Hi,

Currently it's only possible to customize histogram boundaries globally through `ExporterBuilder::with_default_histogram_boundaries`.

It would be great if customizing the boundaries for each specific value recorder was supported.

Something along the following lines
```Rust
let recorder = global::meter("my_meter")
    .f64_value_recorder("my_recorder")
    .with_boundaries(vec![10.0, 50.0, 200.0]) // <-- not possible right now
    .init();
```

Would a contribution be welcome? Is there a workaround to obtain the same behavior (maybe through multiple exporters?)

**Comment by jan-xyz** [comment]
It looks like this will be supported in the future via the `AggregatorSelector`, maybe you can clarify here @TommyCpp?

**Comment by TommyCpp** [comment]
Since the metrics spec has been stabilized. We are looking to migrate the existing implementation onto the stable spec. In the stable metric spec. This can be achieve via [Histogram](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#histogram) instrument and [Histogram aggregation](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#histogram-aggregation) where you can configure your boundaries.

**Comment by Alexx-G** [comment]
@TommyCpp Is there a tracking issue for migration onto the stable spec?
I'm trying to figure out how to set custom buckets for a specific histogram in current (0.16) version and it's not entirely clear, I might be missing something though

**Comment by TommyCpp** [comment]
@Alexx-G I am working on creating some issue to lay out our road map on metrics. It will take some time to implement it. So if many users need those we may explore and see if we can provide some kind of API based on current version.

**Comment by hdost** [comment]
Per the spec this should be customized in the SDK based upon the [Views](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#view)

**Comment by rikonor** [comment]
> So if many users need those we may explore and see if we can provide some kind of API based on current version.

Is this simply not possible currently with `opentelemetry-rust`?

**Comment by jtescher** [comment]
#897 has been released in #1156
